Wearable/src/main/java/com/pimpimmobile/librealarm/WearActivity.java

```diff
@@ -58,25 +64,35 @@
 
     private Vibrator mVibrator;
 
+    private boolean hasRoot = false;
+    private boolean checkedForRoot = false;
```

Comment:
I prefer to have it mHasRoot (for private variables), sScriptsCreated (for static variables) and D (for static final). Just makes it easier to read imo.